buildbot/master/files/config/factories.py

```diff
@@ -90,6 +90,7 @@ def setDefaultWorkdir(self, workdir):
     @defer.inlineCallbacks
     def run(self):
         self.is_windows = re.match('windows.*', self.builder_name) is not None
+        self.is_win_gnu = re.match('windows.*gnu', builder_name) is not None
```

Comment:
Also, with the modification in https://github.com/servo/servo/pull/14651#issuecomment-268398195, these regexes can just become `windows` and `windows-gnu`, as `re.match` only checks the beginning of the string for a match and ignores any further characters.